lib/selections.js

```diff
@@ -152,35 +134,26 @@ export class Selection {
 }
 
 /**
- * Examples:
- * existing: 1-10 | 1-10
- * new_item: 8-12 | 10-15
- * @param {SelectionItem} newItem
- * @param {SelectionItem} existing
+ * @param {MinimalSelectionItem} newItem
```

Comment:
aw man those explanations were good :(

Reply:
you mean the examples? I can add them back, I just thought it made things kind of messy, but if you found them helpful, then they're probably worth keeping

Reply:
for you maybe, but for some1 who hasnt worked on this in the future it's useful

Reply:
okay, sure. I added them back in then :)